src/commands/nodes/node_install.py

```diff
@@ -0,0 +1,189 @@
+import io
+import os
+import platform
+import tarfile
+from os import makedirs
+from pathlib import Path
+
+import click
+import requests
+
+from src.common.utils import greenify
+from src.config.networks import AVAILABLE_NETWORKS
+from src.config.settings import DEFAULT_NETWORK
+from src.nodes.typings import Release
+
+DEFAULT_REQUESTS_TIMEOUT = 60
+
+LATEST_RETH_VERSION = 'v1.5.1'
+LATEST_LIGHTHOUSE_VERSION = 'v7.0.1'
+
+
+@click.option(
+    '--data-dir',
+    default=Path.home() / '.stakewise',
+    envvar='DATA_DIR',
+    help='Path where the nodes data will be placed',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, path_type=Path),
+    show_default=True,
+)
+@click.option(
+    '--network',
+    default=DEFAULT_NETWORK,
+    envvar='NETWORK',
+    help='The network of your nodes.',
+    prompt='Enter the network name',
+    type=click.Choice(
+        AVAILABLE_NETWORKS,
+        case_sensitive=False,
+    ),
+    show_default=True,
+)
+@click.option(
+    '--reth-version',
+    default=LATEST_RETH_VERSION,
+    help='Version of the Reth binary to install.',
+    type=str,
+    show_default=True,
+)
+@click.option(
+    '--lighthouse-version',
+    default=LATEST_LIGHTHOUSE_VERSION,
+    envvar='LIGHTHOUSE_VERSION',
+    help='Version of the Lighthouse binary to install.',
+    type=str,
+    show_default=True,
+)
+@click.command(
+    help='Installs execution node and consensus node to the data dir.',
+)
+def node_install(data_dir: Path, network: str, reth_version: str, lighthouse_version: str) -> None:
```

Comment:
Would be great to check the hardware. Afair min requirements are: 16GB RAM, 2TB SSD storage, modern CPU.